### DISCUSSION TITLE
Change redirect behavior

### DISCUSSION THREAD

**Discussion by hoochalaffa** [YOURLS/YOURLS#2948] [Q&A] [answered]
Hello all. I am not sure what I am looking for except to describe a behavior and try to figure out what I can do, if anything can be done, to address it. I am a layman, please bear with me.

The behavior is that if I change the long url for a link in order to redirect it to a new location, anyone who's already clicked the link will be taken to the old long url. I would like to be able to change the long url and have users directed to the new location.

I understand that the caching of the long url destination on users' web browsers is an expected behavior; I think I read this somewhere. Is there a plugin or setting to get the result of redirecting users to the new long url without also instructing them to clear their browser cache?

**Comment by dgw** [comment, marked as answer]
Try one of the 302-Instead plugins from https://github.com/YOURLS/awesome-yourls#0-9-

Anyone who's already followed the redirect will have to clear their cache. There's no way around that.

**Reply by hoochalaffa** [reply]
thanks much. I didn't have the right vocabulary to find what I was looking for. This helps.
